features/step_definitions/networking.rb

```diff
@@ -805,12 +805,13 @@
   host = node.host
   dhcp_status_timeout = 30
   #Following will take dnsmasq backup and append curl contents to the dnsmasq config after
-  @result = host.exec_admin("cp /etc/dnsmasq.conf /etc/dnsmasq.conf.bak;curl https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/dnsmasq_for_testbridge.conf | sed s/testbr1/#{br_inf}/g | sed s/88.8.8.100,88.8.8.110,24h/#{add_lease}/g >> /etc/dnsmasq.conf;systemctl restart dnsmasq --now")
+  @result = host.exec_admin("cp /etc/dnsmasq.conf /etc/dnsmasq.conf.bak;curl https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/dnsmasq_for_testbridge.conf | sed s/testbr1/#{br_inf}/g | sed s/88.8.8.100,88.8.8.110,24h/#{add_lease}/g > /etc/dnsmasq.conf;systemctl restart dnsmasq --now")
   raise "Failed to configure dnsmasq service" unless @result[:success]
   wait_for(dhcp_status_timeout) {
     if host.exec_admin("systemctl status dnsmasq")[:response].include? "running"
       logger.info("dnsmasq service is running fine")
     else
+      host.exec_admin("cp /etc/dnsmasq.conf.bak /etc/dnsmasq.conf && systemctl restart dnsmasq --now")
```

Comment:
We can use `mv` instead of `cp` here. Faster, and we no longer left a unused .bak file on the system.

Reply:
@liangxia Great observation. mv could have make more sense but such tests are complicated and having a bak file on cluster is safe during this step. This might be mostly required in debugging in case of something exits pre-maturely/crash etc hence needed to be looked into cluster manually in such cases. Having bak file in had would be safe. Anyways such steps will always be followed by ```DHCP service is deconfigured on the "([^"]*) node``` which will eventually remove the bak file so we are good there. We are seeing hiccups in such tests from time to time and such bak file would def be useful sometime.

Reply:
ack